demo/full/scripts/controllers/ContentList.jsx

```diff
@@ -187,6 +192,31 @@ function getIndexOfFirstEnabledContent(contentList) {
   return contentChoiceIndex;
 }
 
+/**
+ * @param {Array.<Object>} drmInfos
+ * @param {Object} fallbacks
+ * @returns {Promise.<Array.<Object>>}
+ */
+function getKeySystemsOption(
+  drmInfos,
+  { fallbackKeyError,
+    fallbackLicenseRequest },
+) {
+  const wantedDRMs = drmInfos
+    .map(drmInfo => ({
+      drm: drmInfo.drm === CUSTOM_DRM_NAME ?
+        drmInfo.customKeySystem :
+        drmInfo.drm,
+      licenseServerUrl: drmInfo.licenseServerUrl,
+      serverCertificateUrl: drmInfo.serverCertificateUrl,
+      fallbackKeyError,
+      fallbackLicenseRequest,
+    }))
+    .filter(drmInfo => drmInfo.drm != null);
```

Comment:
Sure about the` !=` ?

Reply:
It's in the case where `customKeySystem` is not defined